test/integration/targets/pids/tasks/main.yml

```
@@ -0,0 +1,59 @@
+# Test code for the pid module
+# Copyright 2019, Saranya sridharan
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: "Installing the psutil module"
+  pip:
+    name: psutil
+
+- name: "Checking the empty result" 
+  pids: 
+    name: "blahblah"
+  register: emptypids
+
+- name: "Verify that the list of Process IDs (PIDs) returned is empty"
+  assert:
+    that:
+    - emptypids is not changed
+    - emptypids.pids == [] 
+
+- name: "Picking a random process name"
+  command: echo "some-long-random-process-name-{{ 99999999 | random }}"
+  register: random_name
+
+- name: "finding the 'sleep' binary"
+  command: which sleep
+  register: find_sleep
+
+- name: "copying 'sleep' binary"
+  copy:
+    src: "{{ find_sleep.stdout }}"
+    dest: "{{ role_path }}/tasks/{{ random_name.stdout }}"
```

Use the temporary output dir for storing temporary files:

```suggestion
    dest: "{{ output_dir }}/{{ random_name.stdout }}"
```